tests/test_check_function.py

```diff
@@ -23,7 +23,41 @@ def user_data_normalize():
     }
 
 
+@pytest.mark.parametrize(
+    "sct_range, function, index, correct",
+    [
+        ("A1", "SUM", 0, True),      # 1
+        ("A1", "TEST", 0, False),    # 0
+        ("A1", "SUM", 1, True),      # 1
+        ("A1", "SUM", 2, False),     # 0
+        ("A1", "TEST", 1, False),    # 0
+        ("B2", "MEDIAN", 0, True),   # 1
+        ("B2", "MEDIAN", 1, True),   # 1
+        ("B2", "MEDIAN", 2, False),  # 0
+        ("B2", "TEST", 0, False),    # 0
+        ("B2", "TEST", 1, False),    # 0
+        ("B2", "TEST", 2, False),    # 0
+        ("B2", "SUM", 0, False),     # 0
+    ]
+)
+def test_check_function_index(user_data_index, sct_range, function, index, correct):
```

Comment:
This should be moved to be below `# Tests` (above it should be fixtures).

Reply:
Done